.travis.yml

```diff
@@ -1,40 +1,34 @@
 language: node_js
 node_js:
-  - '8'
+- '8'
 env:
   global:
-    # kosssi GITHUB_TOKEN
-    # to generate yours : travis encrypt GITHUB_TOKEN=<your_GITHUB_TOKEN>
-    - secure: "vw+BSo/iU2IpteuzjqdCJsgS6VrlHNc531V51Rqm0AjtZa7LoMn+1xNXL790vs5/LGoXr6TZqZhRuR777V2ASOmyPW5n4ZSPrPzu6e6A4e57brDUuBL/hz5EBXyfyQU/41hU0c8dWJzinEYSDQurt4zQaSlP61nChipuNsx6aXLXQHIQNngCdBUpm4c2KJ7UJy9LpoEVtnxnPt18EvKUsENs3kZraP/twSM2j+n2hKt1In8MRoSZNyNvAwGSI5U+2H3SltKJxn/u0IoPFPoqrbKjbwAhhRAw4xv9X4flSIoxflOZurEUkbKcdLlGPPmRJBLTODq3I9q5yiS5eXcWTfCeniaTHv8KYRrjLuwEb2ZpwRfBgcHqyY4I29yoKNqrfbN5YOgCTLru9tJRT7IvfACRCzK9ToyP3QtUTAqSUfS5CeaWHVmTsWejh072V2kJQRczqhCcS+gCyGVoNU4TY6AysOUP/nZh8H7sQr/mfO/MemufPMtbsmxRH3oY4u4gXLqX0dKcfTimmdcwmLwSPBbUgN80YPR3GVR0zLWltf35Q65P7r3HLkHCfKoO8pGverNkG/izlQYn1muSai9TG0MgaX5eeemwbe5tHmZmXhT1heDFFvzeVAPDeU5OBFG0VjCh6DJcjWDmeVK6HaupEHA06edu4/n2W/uH962cWXk="
-    # Cozy REGISTRY_TOKEN
-    - secure: "CQ44ZVjPAIVRvGgMcduSAX4VmaOrzF3nrJXPFUsNQ2tMVVdaAlI1zTNkdfntqigZgz6Mpk9x+2FcA8qrkNzQ4cW0lPP9f5dBtSH1xetfpXDFKprW4bqzmhgXSXJFTOYxkChOBPmCYJr+Vv4XqDkv/dBEPdfRSgvVyd5YxHSd5HBU91rKyR+Hox7LTnhmlXleLIs11EQgp7RK+L28MBCtCqAa2niFlKKRlqupm2FSpZzxCh9qeY0wKCQzPE+QZUm4waJ1SUGKQwbzAfyOxu2O6rwzaTSknfdJoSX6AMxjRTWBC9c3VfUGM3wjVJIqMhiUj2FQhVIjiDyX/W60/q3UMC5WYzSvEDDfhnKCPwiUKSU7HvSxx9q2YP34evxKo7KPahz9zLlQZvW+GBMyI9hbMDBTALuQVVzhftsvqfrogyYhkEPviiYiVADWNxhacsbUjRA5svXnZvtmCPo7fLFwel/krgC5Z8AwAY0YxWjSHNVScUkieSDCrlxrC09vTtG60unt9gtXbD5ruUkDLWSUwHBv2p0WY1TdP+2Vlf3yYfBvEVbqi59AxG+T17bq7Xv3QZnkrwC+UOCNCeg+gHPv+fRpUaYWzCO4uaoY/nlJa7RulQFhGca3A//6xRo/KgWw9JJnOXJelNjE9oHCUu9xShotUZKOF+O93V/BRqs9ITo="
+  - secure: wqYTIw4e438+O96+WpJKZY25Z6z1S+evb/1e/sz9bN1PZ4bPirgUaLJpqInXS0ohiGH8A6eCVfFUKjqevsVgBa17WZMg74ogcNG71HBSBlKLc8OLxpvB75eqeSC+bN7354KPteCRU1iV4xtYGKKzsy1JEU9HxWVUXZ86BWTAdBxZ6950+jAhhfBUocNDfynL02K7gwjj4TzoKzNDbKwnUu5cxb/rss+m/9Eg+bF8ByLczkzxiNPoGYi0yEfZC+t5Mi9eeNX28ccDDxLJl9wxkAmklpHqWM6AJvNUditCT6xJJ4ApnfvybBNJX7tGELNYsC3SA6b4oigPtssahTN7oWB+zOsOOzcZ5JA8kiPbvjEtbB/jfERyQw+uVoBTZuaWlLWplb8si6wflgPsgzacSVfa3oNtAFSmNaSLkqzBZpUTZbbzm5e1BYQMMvnaQrhChssD3C9LPfCIzofSkHFNMCeDO8Ny5cnbZvwYBnS/5ndBGK0odnyYmYIPBqMT6p3D0+mWyPbrEK2NoZCbIRB1AXfQu92QWkT0DTLLKV1yeCYtvS7pzfg5qR8RJugnbj83GGnoC4gFomGVyC+4hbxN5JAZ0MM2RkYcnlJ1OTC66iSRssUZo/QGuejF6rdFjTNu3ZGfqFLuZkhce/SI0h3Lh++pxk0bN6vBf5M2v2OH2XQ=
```

Comment:
I see you used the `travis encrypt --add` command, this changes the indentation of the .travis.yml file and I am afraid it causes some problems.